hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/Visitor.groovy

```diff
@@ -54,4 +57,7 @@ interface Visitor {
   void visitJob(PigJob job);
   void visitJob(SparkJob job);
   void visitJob(VoldemortBuildPushJob job);
+  void visitJob(HdfsToTeradataJob job);
```

Comment:
For Gobblin, there are more than hundred parameters and I think it's really hard to validate parameters from Hadoop DSL (especially Gobblin is two hops away from Hadoop DSL), and even from Azkaban -- Azkaban only checks some parameters, if a preset is defined. For that reason, there's no visitJob for Gobblin -- it will use visitJob(Job job).